ci-operator/jobs/openshift/knative-serving/openshift-knative-serving-release-next-postsubmits.yaml

```diff
@@ -1,63 +1,5 @@
 postsubmits:
   openshift/knative-serving:
-  - agent: kubernetes
-    always_run: true
-    branches:
-    - ^release-next$
-    cluster: build02
-    decorate: true
-    decoration_config:
-      skip_cloning: true
-    labels:
-      ci-operator.openshift.io/is-promotion: "true"
-      ci-operator.openshift.io/variant: 410-tls
-      ci.openshift.io/generator: prowgen
-    max_concurrency: 1
-    name: branch-ci-openshift-knative-serving-release-next-410-tls-images
```

Comment:
Is this run elsewhere?

Reply:
So far, we had:
```
openshift-knative-serving-release-next__410-tls.yaml
openshift-knative-serving-release-next__411.yaml
openshift-knative-serving-release-next__46.yaml
openshift-knative-serving-release-next__47.yaml
```
IMO, it would be fine to have just 4.6 and 4.11. It depends what you want. Please also see my comment here: https://github.com/openshift/knative-serving/pull/1239#discussion_r965586631

Reply:
I like the idea of running against the old and the newest version more because release-next is supposed to introduce new stuff.